src/components/calendar/calendar.test.jsx

```diff
@@ -155,18 +155,24 @@ describe('<Calendar />', () => {
   });
 
   describe('previous month button', () => {
-    it('displays the previous month', () => {
+    it('displays the previous month when clicked and when enter is pressed', () => {
       render(<Calendar {...requiredProps} />);
       userEvent.click(screen.getByTitle('Change calendar to June 2020'));
       expect(screen.getByText('June 2020')).toBeInTheDocument();
+
+      fireEvent.keyDown(screen.getByTitle('Change calendar to May 2020'), { key: 'Enter', code: 'Enter' });
```

Comment:
```suggestion
      fireEvent.keyDown(screen.getByTitle('Change calendar to May 2020'), { key: 'Enter' });
```

It's recommended to avoid using `code` in favor of the always human readable `key`.